### PR TITLE
Update version and artifact name generated by azure pipeline

### PR DESCRIPTION
**Pull request by bergmania**
###  Goal:
- Have version in the generated artifact name
- Use same pipeline for nightly builds and release builds

###  Details:
 - The new format is:
    - Nightly builds (If source branch is NOT a release format)
        -  e.g Version becomes `9.0.0-preview20210517.63720`  => Artifacts becomes `9.0.0-preview20210517.63720.63720` 
     - Release builds  (If source branch is a release format)
       -  e.g Version becomes `9.0.0-beta003` => Artifacts becomes `9.0.0-beta003.63720` 
- Also adding Azure Pipeline tags to the builds:
-  `Continuous build` vs `Release build`. These can be used to filter releases
image

### NOTE:
- Please `squash and merge` instead of just `merge`. Azure Pipeline manipulations required a couple of attempts 🙈

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/package-lock.json [hunk 1]
```diff
@@ -1842,8 +1842,7 @@
       "version": "1.5.1",
       "resolved": "https://registry.npmjs.org/base64-js/-/base64-js-1.5.1.tgz",
       "integrity": "sha512-AKpaYlHn8t4SVbOHCy+b5+KKgvR4vrsD8vbvrbiQJps7fKDTkjkDry6ji0rUJjC0kzbNePLwzxq8iypo41qeWA==",
```
Comment (AndyButland) [inline src/Umbraco.Web.UI.Client/package-lock.json:1844]: Don't think you intended this file to be checked in?

**Review by AndyButland — APPROVED** [review]
This looks fine to me, and I can see form the build history that the correct names are being created.  It's still in draft so I'll leave it to you to merge when you are happy with it.

File: src/Umbraco.Web.UI.Client/package-lock.json (hunk 1 above)
Comment (bergmania) [reply src/Umbraco.Web.UI.Client/package-lock.json:1844]: No, but Im not really sure why this file is in Git in the first place..

Comment (bergmania) [reply src/Umbraco.Web.UI.Client/package-lock.json:1844]: Then I guess we should just remove from git :)

Comment (nul800sebastiaan) [reply src/Umbraco.Web.UI.Client/package-lock.json:1844]: Haha yes both good pages! ;-) But yes, just do a git delete, all good. 👍

**Pull request merged**